compiler/rustc_mir/src/interpret/validity.rs

```diff
@@ -241,17 +241,18 @@ impl<'rt, 'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> ValidityVisitor<'rt, 'mir, '
             // generators and closures.
             ty::Closure(def_id, _) | ty::Generator(def_id, _, _) => {
                 let mut name = None;
-                if let Some(def_id) = def_id.as_local() {
-                    let tables = self.ecx.tcx.typeck(def_id);
-                    if let Some(upvars) = tables.closure_captures.get(&def_id.to_def_id()) {
+                if let Some(local_def_id) = def_id.as_local() {
```

Comment:
maybe create an issue in project repo saying that this should be more discriptive? i.e. CapturePlace instead of CapturedVar? and then link it as a fixme here.

Reply:
@jgwills can you describe more about the issue here than just like a link. Also don't need name here and if you want to leave names place leave your github ids

Reply:
@jenniferwills